benchmarking/benchmark_framework_module.rb

```diff
@@ -0,0 +1,22 @@
+module BenchmarkFrameworkModule
```

Comment:
What do you think about using the module as a general namespace for your benchmarking framework.

`module BenchmarkFramework`